File: Java/Card.java
package Java;

    public class Card extends Payment{
        Integer idCard;
        Integer number;
        Integer cvv;
        String date;

    public Card(Integer id, Integer idCard, Integer number, Integer cvv, String date){
        super(id);
        this.idCard = idCard;
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}

File: PHP/card.php
<?php
require_once('payment.php');
Class Card extends payment {
    public $idCard;
    public $number;
    public $cvv;
    public $date;

    public function __construct($id, $idCard, $number, $cvv, $date){
        parent::__construct($id);
        $this->idCard = $idCard;
        $this->number = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }
}